styles/scss/globals.scss

```diff
@@ -1,5 +1,9 @@
 // App.scss
 @use "./utils/" as *;
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
```

Comment:
May I ask where you have implemented Tailwind CSS?

Reply:
I tried to use it to before . Now it's deleted.